code/game/objects/structures/crateshelf.dm

```diff
@@ -14,6 +14,14 @@
 	parts = /obj/item/weapon/rack_parts/shelf
 	pass_flags_self = null
 
+/obj/structure/rack/crate_shelf/centcomm_provided/
+    var/how_cheap_percent = 20
+
+/obj/structure/rack/crate_shelf/centcomm_provided/New()
+    . = ..()
+     if(prob(how_cheap_percent))
+          var/osha_violation = TRUE
```

Comment:
```suggestion
          osha_violation = TRUE
```
This var has already been declared at the parent type so you can't redeclare it here

Reply:
Also you cut off a bunch of vars from the parent type. Move your entire code to the blank spot immediately preceeding the "tall" variant.